sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/WebJobsEventHubTestBase.cs

```diff
@@ -73,20 +75,19 @@ protected void ConfigureTestEventHub(IHostBuilder builder)
                         {"AzureWebJobsStorage", StorageTestEnvironment.Instance.StorageConnectionString}
                     });
                 })
-                .ConfigureServices(services =>
-                {
-                    services.Configure<EventHubOptions>(options =>
```

Comment:
I removed this because it seems that we were repeating the same config in the ConfigureDefaultTestHost call below.